src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -224,7 +225,7 @@ internal void ComputeReturnType()
 
             if (_refKind == RefKind.RefReadOnly)
             {
-                DeclaringCompilation.EnsureIsReadOnlyAttributeExists(diagnostics, _syntax.ReturnType.Location, modifyCompilationForRefReadOnly: false);
+                DeclaringCompilation.EnsureIsReadOnlyAttributeExists(diagnostics, _syntax.ReturnType.Location, modifyCompilation: false);
```

Comment:
>EnsureIsReadOnlyAttributeExists [](start = 37, length = 31)

Should we also call EnsureNullableAttributeExists here?

Reply:
Moved from `SynthesizedClosureMethod`.